.changes/next-release/bugfix-2df57233-310e-4788-be6c-f3b37767e60a.json

```diff
@@ -0,0 +1,4 @@
+{
+  "type" : "bugfix",
+  "description" : "Preserve file permissions in the zip archive, fix bugs in gitignore matching and keep mvm and gradle files"
```

Comment:
Is the change user facing, ie is a changelog required here?

Reply:
yep, it does have a user facing change. updated the messaging.